validator/client/validator.go

```diff
@@ -117,21 +119,21 @@ func (v *validator) WaitForActivation(ctx context.Context) error {
 func (v *validator) checkAndLogValidatorStatus(validatorStatuses []*pb.ValidatorActivationResponse_Status) [][]byte {
 	var activatedKeys [][]byte
 	for _, status := range validatorStatuses {
+		pubKey := fmt.Sprintf("0x%s", hex.EncodeToString(status.PublicKey)[:8])
 		log := log.WithFields(logrus.Fields{
-			"pubKey": hex.EncodeToString(status.PublicKey)[:12],
```

Comment:
wont this auto print in the hex format ?

Reply:
it misses `0x` prefix. I'll use Trunc here as well

Reply:
if you want to add `0x` just use `%#x` instead of `0x%s`

Reply:
cool, done.